File: LLM-Translations/GPT-4/avatar/Python/Java/atcoder_ABC042_A.java
import java.util.*;

public class atcoder_ABC042_A {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int[] ABC = new int[3];
        for (int i = 0; i < 3; i++) {
            ABC[i] = sc.nextInt();
        }
        int count = 0;
        int sum = 0;
        for (int num : ABC) {
            if (num == 5) {
                count++;
            }
            sum += num;
        }
        if (count == 2 && sum == 17) {
            System.out.println("YES");
        } else {
            System.out.println("NO");
        }
    }
}
// End of Code

File: LLM-Translations/GPT-4/avatar/Java/C/atcoder_ABC042_A.c
#include <stdio.h>

int main() {
    int x, a, y;
    char h[3] = "NO";

    scanf("%d %d %d", &x, &a, &y);

    if ((x == 5 || x == 7) || (a == 5 || a == 7) || (y == 5 || y == 7)) {
        if (x + y + a == 17) {
            snprintf(h, sizeof(h), "YES");
        }
    }

    printf("%s\n", h);
    return 0;
}

// //End of Code.
